src/services/index.ts

```diff
@@ -1,6 +1,17 @@
 import CachedUserService from './CachedUserService';
 import HttpService from './HttpService';
+import OrganizationUserService from './OrganizationUserService';
+import OrganizationService from './OrganizationService';
 import PreferencesService from './PreferencesService';
 import UserService from './UserService';
 
-export { CachedUserService, HttpService, PreferencesService, UserService };
+export type { Response } from './HttpService';
+
+export {
+  CachedUserService,
+  HttpService,
+  OrganizationUserService,
```

Comment:
nitpick: if these are in alphabetical order, OrganizationService should be before OrganizationUserService

Reply:
thanks, that's how i would usually sort..  in the strings file we are sorting differently.. applied that here. will fix here.

Reply:
```
node -e 'console.log(["OrganizationService", "OrganizationUserService"].sort())'
[ 'OrganizationService', 'OrganizationUserService' ]
```
oh well...